app/angular/src/client/preview/angular/decorators.ts

```diff
@@ -20,3 +23,20 @@ export const moduleMetadata = (metadata: Partial<NgModuleMetadata>) => (storyFn:
     },
   };
 };
+
+export const componentDecorator = (
```

Comment:
we could merge the 2 `componentDecorator` and `templateDecorator` 🤷‍♂️ 

Do not hesitate to suggest a better name 🙏

Reply:
Are there any benefits of keeping these 2 decorators separated? As a user, I think it would be much more understandable if we get a single `componentWrapperDecorator` that can take either a template or a Component as 1st param. What do you think?

Reply:
> Are there any benefits of keeping these 2 decorators separated?
I don't see any. 🚀  I'm merging them together